data/core.yaml

```diff
@@ -801,6 +801,11 @@ en:
     field_source: add source
     field_source_label: Source for {field_name}
     field_source_placeholder: URL, newspaper article, book...
+    source:
+      main_input: General source
+      name: Name of the source
+      url: https://xyz.com
```

Comment:
This is a real website. We should use a fake URL. The [Website](https://github.com/openstreetmap/id-tagging-schema/blob/062459e4b33edfdfeb0df62387c50de43c64dd4d/data/fields/website.json#L8) field uses example.com.